### PR TITLE
Card HTML-CSS Dropdown

### PR DESCRIPTION
**Pull request by ricravenous**
As per design critique, placed the button into the card, next to the header. Moved classes and tags around, along with re-positioning the drop-down navigation to flow over the other card contents.

screen shot 2018-10-16 at 3 49 02 pm
screen shot 2018-10-16 at 3 48 57 pm

Still looking forward to cleaning up for mobile resizing + cleaning up JavaScript targeting.

### CODE REVIEW

**Comment by ricravenous** [comment]
[Integrated Topics Dropdown](https://trello.com/c/KX54pOIu/14-integrated-topics-dropdown)

**Comment by creuter** [comment]
@ricravenous Mind rebasing this before I review?

**Comment by ricravenous** [comment]
Rebased from master, small functionality issues I think a pairing would help iron out:
screen shot 2018-10-16 at 2 47 17 pm
screen shot 2018-10-16 at 2 47 22 pm

File: source/index.html.erb [hunk 1]
```diff
@@ -2,22 +2,23 @@
...
-    <li><a class="topic__link" href="#">Food</a></li>
-  </ul>
-</details>
-
 <div id="card" class="card">
-  <header id="card-topic-header" class="card__category">
-    Ralph's Launch Cards
-  </header>
-
+  <header  class="card__category">
```
Comment (creuter) [inline source/index.html.erb]: It looks like there's no closing `</header>`. Can you remove this tag altogether?

File: source/index.html.erb [hunk 2]
```diff
@@ -2,22 +2,23 @@
...
+   <details class="topics-nav">
+      <summary id="card-topic-header" class="topics-nav__toggle">
+        <h1 id="card-topic-header" class="card__header">
+          Pick a Category
+        <svg class="topics-nav__icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24"><defs><path id="a" d="M0 0h24v24H0V0z"/></defs><clipPath id="b"><use xlink:href="#a" overflow="visible"/></clipPath><path d="M2.5 19h19v2h-19zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 1.82 3.16.77 1.33 1.6-.43 5.31-1.42 4.35-1.16L21 11.49c.81-.23 1.28-1.05 1.07-1.85z" clip-path="url(#b)"/><path clip-path="url(#b)" fill="none" d="M0 0h24v24H0V0z"/></svg>
+        </h1>
+     </summary>
+      <ul class="topics-nav__menu">
```
Comment (creuter) [inline source/index.html.erb]: Indentation is still inconsistent throughout this file. It looks like there are a few ways you can have Sublime do the indentation clenaup for you: http://sweetme.at/2013/10/04/how-to-automatically-reindent-code-with-sublime-text/

File: source/stylesheets/header.scss [hunk 3]
```diff
@@ -1,49 +1,77 @@
...
+  background-color: $blue-fill;
+  border-radius: $base-border-radius;
+  border: $base-border;
+  box-sizing: border-box;
+  color: $white;
+  display: inline-flex;
+  font-size: $h1-font-size;
+  font-weight: $h1-font-weight;
+  margin: 0 auto;
+  padding-left: auto;
```
Comment (creuter) [inline source/stylesheets/header.scss]: These `padding-left` and `padding-right` values probably aren't doing anything, and are being overridden by the padding declaration just below, regardless.

**Review by creuter — APPROVED** [review]
Nice progress 🚢

**Pull request merged**